x-pack/plugins/ml/public/application/services/new_job_capabilities/new_job_capabilities_service.ts

```diff
@@ -16,7 +16,12 @@ import {
 import { ml } from '../ml_api_service';
 import { processTextAndKeywordFields, NewJobCapabilitiesServiceBase } from './new_job_capabilities';
 
-const categoryFieldTypes = [ES_FIELD_TYPES.TEXT, ES_FIELD_TYPES.KEYWORD, ES_FIELD_TYPES.IP];
+const categoryFieldTypes = [
```

Comment:
Should I be able to do `distinct_count` on `version` fields. In the wizards I can only select `keyword` types and not `version`.

Reply:
yes, good point.
Should we treat `version` fields exactly the same as `keyword` fields?